header/js/typeahead.js

```diff
@@ -99,4 +107,23 @@ function trackSearchEvent (context) {
 	document.body.dispatchEvent(tracking);
 }
 
+// This is purposefully quite rubbish to be small, fast and target only the most-problematic chars.
+function latinize (text) {
+	const replacements = {
+		'a': ['à', 'á', 'â', 'ä', 'ã', 'å'],
```

Comment:
I think there is... if we return the `searchLabel` field (which is normalized) alongside the `prefLabel` we can expose this to the search suggestions component but display the latter to the user. Something to tackle another day I reckon.